flatpak_builder_lint/checks/finish_args.py

```diff
@@ -31,6 +31,10 @@ def _validate(self, appid: Optional[str], finish_args: dict[str, Set[str]]) -> N
                 if fs.startswith(f"{xdg_dir}/") and fs.endswith(":create"):
                     self.errors.add(f"finish-args-unnecessary-{xdg_dir}-access")
 
+            for fs in finish_args["filesystem"]:
+                if fs.startswith("/home"):
```

Comment:
and `/var/home`. I can see that as a broken workaround.

Reply:
```
>>> fs = "/var/home"
>>> fs.startswith("/home")
False
```

Reply:
Done.